cypress/e2e/apply/lawyers/prevent-signups.feature

```diff
@@ -1,37 +1,57 @@
-Feature:
-  I want to apply to be added to the ‘Find a lawyer abroad’ service but the list does not exist.
+Feature: I want to apply to be added to the ‘Find a lawyer abroad’ service but the list does not exist.
```

Comment:
Does this need to be 
1. renamed to something more generic
2. moved to a different directory